system_tests/espresso_streamer_test.go

```diff
@@ -2,110 +2,91 @@ package arbtest
+	transactionBytes, err := hex.DecodeString(hexVal)
+	if err != nil {
+		log.Crit("Failed to decode hex string", "err", err)
+	}
 
+	data = append(data, TestBlock{
+		blockNumber: 1,
+		transactionsInBlock: espressoClient.TransactionsInBlock{
+			Transactions: []types.Bytes{transactionBytes},
+		},
+	})
+	return data
+}
 
-func TestShouldPopMessagesInOrder(t *testing.T){
-  ctx, cancel := context.WithCancel(context.Background())
+func TestShouldPopMessagesInOrder(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-  log.Info("Test is running")
-
-  mockParentChainClient := new(mockParentChainClient)
-  mockEspressoClient := new(mockEspressoClient)
-  
-  //simulate the call to the tee verifier returning a byte array. To the streamer, this indicates the attestation quote is valid.
-  mockParentChainClient.On("CallContract").Return([]byte{1}, nil)
-  // create a new streamer object
-  streamer := espressostreamer.NewEspressoStreamer(1, []string{""}, 1, time.Millisecond, time.Millisecond, common.Address{}, mockParentChainClient, mockEspressoClient)
-  // Get the data for this test 
-  testBlocks := ShouldPopMessagesInOrderData()
-  
-  mockEspressoClient.On("FetchLatestBlockHeight", ctx).Return(testBlocks[0].blockNumber, nil)
-  mockEspressoClient.On("FetchTransactionsInBlock", ctx, testBlocks[0].blockNumber, uint64(1)).Return(testBlocks[0].transactionsInBlock, nil)
-  // manually crank the streamers polling function to read an individual hotshot block prepared for the mockEspressoClient
-  err := streamer.QueueMessagesFromHotshot(ctx)
-  Require(t, err)
-
-  msg, err := streamer.Next()
-  //assert we did not have an error on next
-  Require(t, err)
-  //assert that the streamer believe this message to have originated at hotshot height 1
-  assert.Equal(t, msg.HotshotHeight, 1)
+	log.Info("Test is running")
+
+	mockEspressoClient := new(mockEspressoClient)
+	mockEspressoTEEVerifierClient := new(mockEspressoTEEVerifier)
+
+	//simulate the call to the tee verifier returning a byte array. To the streamer, this indicates the attestation quote is valid.
+	mockEspressoTEEVerifierClient.On("Verify", mock.Anything, mock.Anything, mock.Anything).Return(nil)
+	// create a new streamer object
+	streamer := espressostreamer.NewEspressoStreamer(1, 1, time.Millisecond, time.Millisecond, mockEspressoTEEVerifierClient, mockEspressoClient)
+	streamer.Reset(735805, 1)
```

Comment:
Whats with the Reset to the random number here?

Reply:
Is just the data returns 735805 as the first pos thats why